exporter.go

```diff
@@ -72,9 +72,10 @@ type BuildReport struct {
 }
 
 type ImageReport struct {
-	Tags    []string `toml:"tags"`
-	ImageID string   `toml:"image-id,omitempty"`
-	Digest  string   `toml:"digest,omitempty"`
+	Tags         []string `toml:"tags"`
+	ImageID      string   `toml:"image-id,omitempty"`
+	Digest       string   `toml:"digest,omitempty"`
+	ManifestSize string   `toml:",manifest-size,omitempty"`
```

Comment:
```suggestion
	ManifestSize int64   `toml:"manifest-size,omitempty"`
```
Why not just make this an `int64` instead of converting to a string?

Reply:
You're totally right :) I'll make this change. Thanks!

Reply:
If this is `int64`, will we be able to "unset" it for older platform APIs?

Reply:
I think that if it's set to `0`, it omitted from `report.toml` thanks to the `omitempty` flag.

Reply:
I added a print of `report.toml` after it's written and you were right - `manifest-size` is printed even if it's `0` and `PlatformAPI=0.4`:
```
Setting CNB_PLATFORM_API=0.4
...
content: [build]
[image]
  tags = ["yaelharel/test-app:my-tag"]
  digest = "sha256:b5a675383bcec612389b81bd5f70772adb452706d014965c19685e94480c77e4"
  manifest-size = 0
```

This is weird because the following code outputs `{}`:
```
package main
import (
	"encoding/json"
	"fmt"
)
type MyStr struct {
	Num int64 `json:",omitempty"`
}
func main() {
	x := MyStr{
		Num : 0,
	}
	j, _ := json.Marshal(x)
	fmt.Println(string(j))
}
```

Maybe `Marshal` works differently than `lifecycle.WriteTOML`.

Reply:
So first I worked with `json` instead of `toml`, but the way to solve it is with `omitzero` instead of `omitempty`.

Reply:
With `omitzero`:
```
Setting CNB_PLATFORM_API=0.4
...
*** Digest: sha256:c0885c0619709bfc53731570359dd553fb6e36d2493d757d51f304082ba80616

*** Manifest Size: 3185
content: [build]

[image]
  tags = ["yaelharel/test-app2:my-tag2"]
  digest = "sha256:c0885c0619709bfc53731570359dd553fb6e36d2493d757d51f304082ba80616"
```

In this example `manifest-size` doesn't exist in `report.toml` since `PlatformAPI` is `0.4`.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/069147f1c476abbe01ee8c1b9a044d34ac8cafa1